src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -2106,10 +2174,11 @@ private function getTrustedValues($type, $ip = null)
     {
         $clientValues = [];
         $forwardedValues = [];
+        $defaultIp = '0.0.0.0';
```

Comment:
should be reverted: this added variable provides nothing in practice

Reply:
yes, it does provide. a code must never use a hardcoded repeating string - at least such string must be placed in a private constant; but, as your code does not use private constants, as far as I can see, I decided to assign it to a variable inside the method.